client/src/main.ts

```diff
@@ -1,15 +1,16 @@
 import Vue from 'vue'
 import App from './App.vue'
-import router from './router'
+import { getVueRouter } from '@/router'
 import store from './store'
 import { getConfig } from '@/plugins/getConfig'
-import vuetify from './plugins/vuetify'
+import Vuetify from "vuetify/lib"
```

Comment:
This is replacing a plugin with a direct Vuetify import...
1. you should delete that obsolete plugin
2. you should copy over the other Vuetify override properties

... Or change it back and just update the iconfont property.

Reply:
Absolutely, will do! I'll look into removing it.

I saw it imported directly into some components so i hesitated in doing so. I don't know the whole flow and how to test it... so i didn't want to do too much that could potentially be destructive to unknown parts.

Reply:
Weird. It should not be needed in other components if it's already loaded when Vue is started.

Reply:
Is there a way to test the other 4 components that import the vuetify plugin and confirm that they don't need to?

Reply:
I'm not too familiar with those components tbh. I think i'd prefer to defer the refactoring ( ie removal of the plugin etc ) to the style tickets. Are you ok with that?

Reply:
Yep.

Reply:
So what are you doing for the icons? Going MDI or MD?

Reply:
Right now, using them both?
We should probably convert them all over to MDI as that is what the other UI's/designs so on, just don't want go out of the scope of this ticket. ie refactoring the style/icons/btns etc in this ui.

Reply:
So it works if you import both but start Vuetify with the old ones?

https://github.com/bcgov/namerequest/blob/9bf8cc015d51b94be520d524c7aeaed09576a659/client/src/plugins/vuetify.ts#L24

Reply:
Yea, both icon types are displaying at the moment. I've tried to remove MD but we will need to go through and find the equivalent in MDI and refactor/rename them. (In the style update tickets ;) )

Reply:
I'm still nervous about this. I see you do instantiate Vuetify with "mdi" iconfont (main.ts line 40), and then some components also instantiate Vuetify with "md" iconfont (via plugs/vuetify.ts). This suggests some part of the app have one font and other parts have another. I'm not sure how that plays together 😨

Reply:
I agree completely, this needs some attention, any proposals as to what to do?

This ticket was to bring in the SbcHeader and allow for authentication. Obviously after importing it, the header inherited style from the UI that skewed it's design badly and the current Icon library didn't support the Common Components requirements.

We can ..
A: Take this to the team and ask to extend this tickets scope and requirements to include refactoring the base Sass to bring it in line with the other UI's.
B: Tackle this work in a future style update ticket, under this story..
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5467
C: any other suggestions i'm open too!

Reply:
Can you JUST change to using MDI icons (and revert the Vuetify code change you made)?

Reply:
Yep

Reply:
Just going to add in a little extra override style to the header as well 👍 

I viewed the style changes as a whole a bit larger scope than just the icons, my thoughts is they were creating the epic i linked above to capture all this work, which is i pushed back on just trying to add onto this ticket.

Reply:
Do you need to change any icon declarations? (Eg, v-icon from "close" to "mdi-close" ?)

Reply:
Not if we keep the plugin, it seems. Just had to change the `iconfont` in the plugin.